react-frontend/src/sharedComponent/EmployeeNameAndImageComponent.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import ImageCoreComponent from "../coreComponents/ImageCoreComponent";
+import LabelCoreComponent from "../coreComponents/LabelCoreComponent";
+const EmployeeNameAndImageComponent = ({
```

Comment:
whenever there is `and` in file name, either your file is not following single responsibility principle or you have named file wrong